### PR TITLE
Automated PID value orchestration

### PR DESCRIPTION
**Pull request by oldgalileo**
This branch is designed to implement a simpler PID tuning system. The only action required on the developer is to create a PID Container with a map of all their values. The container orchestrator will proceed to manage the adding of all the PID values to the dashboard, which it does through a dashboard agnostic interface. An example is used in `Robot.java`, called `SmartDashboardModifierImpl`, which implements it for SmartDashboard.

### CODE REVIEW

**Review by leijurv — APPROVED** [review]
Looks good !

**Comment by oldgalileo** [comment]
As with `align-trim`, the code here was designed with a future merge into standard in mind. For that reason, it is extensible and generic and can be easily moved into our standard library. This motivated some of the design choices which, at first glance, may seem overkill.

File: src/org/usfirst/frc4904/robot/Robot.java [hunk 1]
```diff
@@ -55,7 +69,9 @@ public void alwaysExecute() {
 	public void disabledInitialize() {}
 	
 	@Override
-	public void disabledExecute() {}
+	public void disabledExecute() {
+
```
Comment (carturn) [inline src/org/usfirst/frc4904/robot/Robot.java]: Please remove empty line at some point.

**Comment by oldgalileo** [comment]
See #17 and #18

**Pull request closed without merging**